### ISSUE TITLE
npm run tauri dev failed 

### ISSUE THREAD

**Issue by ghost** [rome-user/tauri-clojurescript-template#1]
Here's what I got:
```
> tauri-app@0.0.0 tauri
> tauri dev

/home/a/Documents/ecchiman/node_modules/@tauri-apps/cli/index.js:250
    throw loadError
    ^

Error: Cannot find module '@tauri-apps/cli-linux-x64-gnu'
Require stack:
- /home/a/Documents/ecchiman/node_modules/@tauri-apps/cli/index.js
- /home/a/Documents/ecchiman/node_modules/@tauri-apps/cli/main.js
- /home/a/Documents/ecchiman/node_modules/@tauri-apps/cli/tauri.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1070:15)
    at Module._load (node:internal/modules/cjs/loader:923:27)
    at Module.require (node:internal/modules/cjs/loader:1137:19)
    at require (node:internal/modules/helpers:121:18)
    at Object.<anonymous> (/home/a/Documents/ecchiman/node_modules/@tauri-apps/cli/index.js:190:31)
    at Module._compile (node:internal/modules/cjs/loader:1255:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1309:10)
    at Module.load (node:internal/modules/cjs/loader:1113:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Module.require (node:internal/modules/cjs/loader:1137:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/a/Documents/ecchiman/node_modules/@tauri-apps/cli/index.js',
    '/home/a/Documents/ecchiman/node_modules/@tauri-apps/cli/main.js',
    '/home/a/Documents/ecchiman/node_modules/@tauri-apps/cli/tauri.js'
  ]
}

Node.js v20.3.0

```

**Comment by ghost** [comment]
This is solved by `npm install --save-dev @tauri-apps/cli`, could you please add it to the README?

**Comment by rome-user** [comment]
Hi, thanks for reporting. I am not sure why installing `@tauri-apps/cli` is necessary when the package.json already lists it.
https://github.com/rome-user/tauri-clojurescript-template/blob/d9e9230b688b112a3cd34571737cc44896b343dc/package.json#L12

Assuming you followed instructions, this is maybe package-lock.json issue. I'll investigate.

**Comment by rome-user** [comment]
After taking a look at the package-lock.json, I noticed the following.

https://github.com/rome-user/tauri-clojurescript-template/blob/d9e9230b688b112a3cd34571737cc44896b343dc/package-lock.json#L50-L61

While the Mac dependency is installed, the rest of them are not. I've patched the package-lock.json file and tested the README instructions in an Alpine Linux container. Re-open the issue if the problem isn't fixed on your end.
